packages/trackerless-network/src/logic/protocol-integration/stream-message/StreamMessageTranslator.ts

```diff
@@ -123,17 +150,17 @@ export class StreamMessageTranslator {
         if (msg.newGroupKey) {
             newGroupKey = new OldEncryptedGroupKey(
                 msg.newGroupKey!.groupKeyId,
-                msg.newGroupKey!.encryptedGroupKeyHex,
-                msg.newGroupKey!.serialized
+                toUTF8(msg.newGroupKey!.data),
             )
         }
         const translated = new OldStreamMessage<T>({
-            signature: msg.signature,
+            signature: toUTF8(msg.signature),
```

Comment:
Should be `toHex`? Also at line `153` and e.g. in `utils.ts:9`

Maybe should double-check cases when we convert from/to a binary id like this? Usually that binary is just bytes which we represent in hex, not UTF8 chars.